.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+dist: trusty
+
+language: python
+python: 3.5
+
+install:
+  # Install code sniffer.
+    pip install pycodestyle;
```

Comment:
And keep only 2 spaces and then the `-`, then another space and the the command.
Example:
```yml
  - somethin
```